docs/program/sdks/_index.md

```diff
@@ -204,6 +205,43 @@ async function main() {
 main();
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+```cpp {class="line-numbers linkable-line-numbers"}
```

Comment:
nit: `cpp` doesn't appear to highlight, and stands out as missing against the other three. Seems to be [the right one](https://gohugo.io/content-management/syntax-highlighting/) though (or maybe `c++` ?). I can investigate.

Reply:
I created https://github.com/viamrobotics/docs/pull/1268 which I believe adds support for C++ highlighting. Let me know if I missed something!

Reply:
thank you!